stage1/init/common/pod.go

```diff
@@ -298,26 +394,54 @@ func appToSystemd(p *stage1commontypes.Pod, ra *schema.RuntimeApp, interactive b
 		env.Set("AC_METADATA_URL", p.MetadataServiceURL)
 	}
 
-	if err := writeEnvFile(p, env, appName, privateUsers); err != nil {
-		return errwrap.Wrap(errors.New("unable to write environment file"), err)
+	// TODO(iaguis): make enterexec use the same format as systemd
```

Comment:
Probably not a lot, but I'd like to get this in a mergeable state and then fix it to make sure we make it to the next release.